docs/sources/operators-guide/mimir-runbooks/_index.md

```diff
@@ -1388,6 +1388,22 @@ Mimir has a limit on the query length.
 This limit is applied to partial queries, after they've split (according to time) by the query-frontend. This limit protects the system’s stability from potential abuse or mistakes.
 To configure the limit on a per-tenant basis, use the `-store.max-query-length` option (or `max_query_length` in the runtime configuration).
 
+### err-mimir-max-total-query-length
+
+This error occurs when the time range of a query exceeds the configured maximum length. For a limit on the partial (after sharding) query length, see [err-mimir-max-query-length](#err-mimir-max-query-length).
+
+Both PromQL instant and range queries can fetch metrics data over a period of time.
+A [range query](https://prometheus.io/docs/prometheus/latest/querying/api/#range-queries) requires a `start` and `end` timestamp, so the difference of `end` minus `start` is the time range length of the query.
+An [instant query](https://prometheus.io/docs/prometheus/latest/querying/api/#instant-queries) requires a `time` parameter and the query is executed fetching samples at that point in time.
```

Comment:
The new limit (and so error) applies only to range queries, isn't it?

Reply:
yes , I quickly tested in a local mimir install. Updating docs one more time.

Reply:
updated docs, please re-review the doc parts + changelog, thank you